app/src/main/java/ch/epfl/sweng/favours/FavoursFragment.java

```diff
@@ -52,20 +53,23 @@ public void afterTextChanged(Editable editable) {
         binding.addFavor.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                if (title.get().isEmpty()) {
-
-                    CharSequence text = "You cannot create favor with an empty title!";
-                    int duration = Toast.LENGTH_SHORT;
-                } else if (description.get().isEmpty()) {
-                    //Error cannot create favor with empty description
+                if (newFavor.get(Favor.StringFields.title).isEmpty() || newFavor.get(Favor.StringFields.description).isEmpty()) {
+                    Log.d(TAG, "Failure to check favor input");
+                    launchToast("There is an empty field left");
```

Comment:
I'll update in the next sprint to check individually for all fields and provide descriptive toasts to the user